### ISSUE TITLE
Lines created in gradient

### ISSUE THREAD

**Issue by beingmrkenny** [LeaVerou/conic-gradient#17]
I've thrown some dynamically generated stops at this and it's handled everything well apart from a few of the stops. With the stops below, there are two white lines at roughly the 11 o'clock and 5 o'clock positions.

Having looked into it, I think it's specific to the colours themselves, rather than my massively over-precise percentages. I've tweaked two of the colours between 'broken' and 'fixed', and this workaround seems to fix it.

```
var brokenStops = [
    '#110f3e 0%',
    '#110f3e 14.775462962962962%',
    '#224473 19.091435185185183%',
    '#2f8cb8 22.15625%',
    '#d6f622 22.15625%',
    '#fffcb3 22.453703703703702%',
    '#41c1de 22.453703703703702%',
    '#3f8cb7 36.123703703703704%',
    '#3f8cb7 74.10199074074075%',
    '#1c6790 87.77199074074075%',
    '#fb6ff2 87.77199074074075%',
    '#ea2d28 88.06828703703704%',
    '#6a4d9c 88.06828703703704%',
    '#34326f 91.13425925925925%',
    '#110f3e 95.45023148148148%',
    '#110f3e 100%'
];

var fixedStops = [
    '#110f3e 0%',
    '#110f3e 14.775462962962962%',
    '#224473 19.091435185185183%',
    '#2f8cb8 22.15625%',
    '#d6f622 22.15625%',
    '#fffcb3 22.453703703703702%',
    '#41c1de 22.453703703703702%',
    '#41c1de 36.123703703703704%', // changed
    '#3f8cb7 74.10199074074075%',
    '#1c6790 87.77199074074075%',
    '#fb6ff2 87.77199074074075%',
    '#ea2d28 88.06828703703704%',
    '#6a4d9c 88.06828703703704%',
    '#34326f 91.13425925925925%',
    '#110f3e 95.45023148148148%',
    '#110f3d 100%'                // changed
];

var broken = new ConicGradient({
    stops: brokenStops.join(','),
    repeating: false,
    size: 500
});

var fixed = new ConicGradient({
    stops: fixedStops.join(','),
    repeating: false,
    size: 500
});
```

Broken:
![broken](https://cloud.githubusercontent.com/assets/2005857/17244936/1bc7671a-557c-11e6-84f5-f1abb4e73787.png)

Fixed:
![fixed](https://cloud.githubusercontent.com/assets/2005857/17244937/1bcc7034-557c-11e6-809f-741924fad460.png)

I checked it with these truncated percentages, and I still get the white lines as above.

```
var brokenStops = [
    '#110f3e 0%',
    '#110f3e 14.77%',
    '#224473 19.09%',
    '#2f8cb8 22.15%',
    '#d6f622 22.15%',
    '#fffcb3 22.45%',
    '#41c1de 22.45%',
    '#3f8cb7 36.12%',
    '#3f8cb7 74.10%',
    '#1c6790 87.77%',
    '#fb6ff2 87.77%',
    '#ea2d28 88.06%',
    '#6a4d9c 88.06%',
    '#34326f 91.13%',
    '#110f3e 95.45%',
    '#110f3e 100%'
];
```

**Comment by beingmrkenny** [comment]
This won't help, but last night it was also putting a line at the 0º position, between the two #110f3e stops at the start and end of the gradient. This morning it doesn't seem to be doing that anymore, but I'm not sure what changed :)

**Comment by eonj** [comment]
@beingmrkenny I think this bug is resolved via #19, so @LeaVerou closed this. Can you test it again in your own?

**Comment by beingmrkenny** [comment]
Yes, seems to be fixed!

Test files attached:

[Archive.zip](https://github.com/LeaVerou/conic-gradient/files/686091/Archive.zip)
